ckanext/xloader/plugin.py

```diff
@@ -104,6 +137,11 @@ def notify(self, entity, operation):
     # IResourceController
 
     def after_resource_create(self, context, resource_dict):
+        if utils.requires_successful_validation_report():
+            log.debug("Skipping xloading resource %s because the "
```

Comment:
Perhaps, for clarity, this should say "Deferring" rather than "Skipping".

Reply:
Done!